mycpp/gc_builtins.cc

```diff
@@ -4,13 +4,17 @@
 
 // Translation of Python's print().
 void print(Str* s) {
-  fputs(s->data(), stdout);
+  for (int i = 0; i < len(s); ++i) {
+    fputc(s->data()[i], stdout);
+  }
   fputs("\n", stdout);
 }
 
 // Like print(..., file=sys.stderr), but Python code explicitly calls it.
 void println_stderr(Str* s) {
-  fputs(s->data(), stderr);
+  for (int i = 0; i < len(s); ++i) {
```

Comment:
Ah I see, I guess this was necessary to make tests pass with NUL bytes!


When we do the C I/O -> POSIX I/O changes, I think this will end up using `man 2 write`

```
write(0, s->data(), len(s))
write(0, "\n", 1)
```